internal/backend/dispatcher/dispatcher.go

```diff
@@ -90,8 +91,8 @@ func (d *dispatcher) Redispatch(ctx context.Context, eventID sdktypes.EventID, o
 }
 
 func (d *dispatcher) Start(context.Context) error {
-	w := worker.New(d.temporal.Temporal(), taskQueueName, worker.Options{})
-	w.RegisterWorkflow(d.eventsWorkflow)
+	w := worker.New(d.temporal.Temporal(), sdktypes.TaskQueueName, worker.Options{})
+	w.RegisterWorkflowWithOptions(d.eventsWorkflow, workflow.RegisterOptions{Name: sdktypes.EventsWorkflow})
```

Comment:
i think you should have a new workflow, not reuse the events workflow.

Reply:
why? IMO it's actually an event workflow which is triggered by temporal and not, e.g. curl. I wanted it to be exactly as normal events workflow. Moreover in temporal UI it make a lot of sense. I could trigger code by curl, or use scheduler as a trigger, or even trigger scheduler workflow manually and all of above will look the same in UI, which is perfect